plugins/plugins.go

```diff
@@ -10,13 +10,23 @@ import (
 	"go.uber.org/zap/zapcore"
 )
 
-type EnvConfig interface {
+// LoggingConfigurer controls static logging related configuration that is inherited from the chainlink application to the
+// given LOOP executable.
+type LoggingConfigurer interface {
 	LogLevel() zapcore.Level
 	JSONConsole() bool
 	LogUnixTimestamps() bool
 }
 
-func SetEnvConfig(cmd *exec.Cmd, cfg EnvConfig) {
+// EnvConfigurer is the configuration interface between the application and the LOOP.
+// It separates static and dynamic configuration. Logging configuration can and is inherited statically while the
+// port the the LOOP is to use for prometheus, which is created dynamically at run time the chainlink Application.
+type EnvConfigurer interface {
```

Comment:
Why the rename? These types are still just very plain interfaces. WDYT about favoring docs on the Set/Get funcs instead?